.travis.yml

```diff
@@ -45,7 +45,7 @@ script:
   # Install Janus
   - curl -sL https://raw.githubusercontent.com/ethereumproject/janus/master/get.sh | bash
   - export PATH=$PATH:$PWD/janusbin
-  - export APP_VERSION_GIT_TAG="$(janus version -format TAG_OR_NIGHTLY)"
+  - export APP_VERSION_GIT_TAG="$(janus version -format 'v%M.%m.%C-%S')"
```

Comment:
> As we're uploading _ONLY_ from master, so it works for us.

OK. In order to play it safe, I would recommend not tagging merge commits. I must say, too, that I still have my doubts about the consistency of it, since:
- it appears that the count depends also "active side branches," which I don't completely understand why
- I've seen inconsistent counts twice with geth now, with AppVeyor yielding different results than Travis. I think both of those times were from tagged (eg. `--branch=v3.5.0`) commit builds

---- 

Re `%B`: Very interesting. I'll be tinkering with Janus today and will keep you posted.